src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -374,19 +374,21 @@ uint32_t ResumptionDataDB::GetGlobalIgnOnCounter() const {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock autolock(resumption_lock_);
 
-  uint32_t global_ign_on_counter = 0;
-
   utils::dbms::SQLQuery query(db());
   if (query.Prepare(kSelectGlobalIgnOnCounter)) {
     if (query.Exec()) {
-      global_ign_on_counter = query.GetUInteger(0);
+      const auto global_ign_on_counter = query.GetUInteger(0);
       LOG4CXX_DEBUG(logger_,
                     "Global Ign On Counter = " << global_ign_on_counter);
       return global_ign_on_counter;
+    } else {
+      LOG4CXX_ERROR(logger_,
+                    "Problem with exec query : " << kSelectGlobalIgnOnCounter);
```

Comment:
This else case is already covered, this will just result in this log being logged twice

Reply:
@jacobkeeler fixed 24e9b7cd8e